presto-main/src/main/java/io/prestosql/sql/gen/OrderingCompiler.java

```diff
@@ -248,7 +248,7 @@ private PageWithPositionComparator internalCompilePageWithPositionComparator(Lis
             comparator = pageWithPositionsComparatorClass.getConstructor().newInstance();
         }
         catch (Throwable t) {
-            log.error(t, "Error compiling merge sort comparator for channels %s with order %s", sortChannels, sortChannels);
+            log.error(t, "Error compiling comparator for channels %s with order %s", sortChannels, sortChannels);
```

Comment:
`Fix error message` commit

Please squash this with the first commit which broadens the use of OrderingCompiler. Otherwise you lose the context of the "fix".